### ISSUE TITLE
Updating pylabrad with pip

### ISSUE THREAD

**Issue by zibrov-zlobin** [labrad/pylabrad#203]
It seems that the package in pip doesn't have the latest changes to the datavault. 
had to do the following to update:
```
 pip uninstall pylabrad
 pip install git+git://github.com/labrad/pylabrad.git
```

**Comment by DanielSank** [comment]
Ok, thanks. I wonder if we can setup a hook to update pypi when we tag a commit.

**Comment by jwenner** [comment]
Can we now close this issue as PyPi now has a recent release?

**Comment by maffoo** [comment]
Yeah, should have been closed when #204 got merged.
